rubicon/objc/runtime.py

```diff
@@ -664,11 +664,39 @@ class objc_super(Structure):
 
 
 # http://stackoverflow.com/questions/3095360/what-exactly-is-super-in-objective-c
-def send_super(receiver, selName, *args, **kwargs):
+def send_super(cls, receiver, selName, *args, **kwargs):
     """Send a message named selName to the super of the receiver.
 
     This is the equivalent of [super selname:args].
+
+    Since proper 'super' semantics requires lexical scoping, the cls argument is now required.
+    It basically instructs this runtime what the lexical scope was of the caller, so that it
+    may find the appropriate superclass method implementation (if any) to call. (See issue #107)
+
+    As such, cls can be a string, which is the class name of the lexical scope of the caller,
```

Comment:
I'm still not sure when using a string for `cls` would be needed. In all normal use cases, you'll pass `__class__` as the first argument. If for whatever reason that doesn't work, you can use `ObjCClass` explicitly to get a class with a specific name.

Reply:
Yeah you're right.  Will fix.

Man your Python Kung Fu is strong.  Thanks for taking the time to explain this to me.  Like I said, I'm a C++/C/Obj-C guy -- Python is relatively new to me (really only learned it properly for this project I'm working on).

Very cool! Thanks!